### DISCUSSION TITLE
Template Repository to Monorepo

### DISCUSSION THREAD

**Discussion by ThallyssonKlein** [community/community#23795] [New to GitHub]
<p>I made a simple template repository. My doubt is, it’s possible to use this template inside another already created repository (monorepo)?</p>
<p>Like this:</p>
<p>ThallyssonKlein/MyTemplate<br>
ThallyssonKlein/SomeMonorepo/SomeProjectsInsideMonorepo(Use template here)</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>I’m afraid the current template feature is intended for a polyrepo model of development, rather than a monorepo model, sorry!</p>
<p>We’re always working to improve GitHub, and we consider every suggestion we receive, so perhaps you’d like to submit a feature request about templating inside of a monorepo <a href="https://support.github.com/contact/feedback">through our official product feedback form</a> so that our product team can see exactly how you’d like GitHub to work?</p>
